recipes/unittest2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "unittest2" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "22882a0e418c284e1f718a822b3b022944d53d2d908e1690b319a9d3eb2c0579" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - no-argparse-req.patch
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python <3
+    - pip
+    - six
+    - traceback2
+  run:
+    - python <3
+    - six
+    - traceback2
+
+test:
+  imports:
+    - unittest2
+
+about:
+  home: https://pypi.org/project/unittest2/
+  license: BSD
```

Comment:
What version of BSD?

Reply:
Oh hmmm well bad news. I wrote BSD because that's what PyPI lists, which is also what `setup.py` lists. But the text in `__init__.py` is something entirely nonstandard, as far as I can tell.

In this case I think the things that say BSD capture authorial intent better; it looks like the `__init__.py` header may have been copy/pasted without really thinking through its implications. There's a link in `setup.py` that implies that the intended license is the 3-clause BSD.

Reply:
Reading the license text, specifically, "This module is free software, and you may redistribute it and/or modify
it under the same terms as Python itself", it sounds like this is PSF 2 actually.

Though there is some text at the end stating there is no warranty, which is similar to BSD/MIT/ISC, but not really the same as any of those. Not sure what we ant to say about that if anything.

Reply:
Yeah, but then check out the top of [setup.py](https://hg.python.org/unittest2/file/tip/setup.py) as well as the OSI license specified. It's just not self-consistent but, in my interpretation, writing the `setup.py` is probably when the author is going to be most actively thinking about what they want for licensing. (Although the author given in the setup.py header comments and the setuptools metadata are different too ...)

Reply:
Overall I'd lean towards saying that the intention is for standard BSD 3-clause licensing. If that's the case, do we need to bundle a LICENSE.txt?

Reply:
We could do `PSF 2, BSD`. WDYT?

Reply:
Seems as good as any other option. I'll also copy in the text of the Voidspace BSD license that's linked in the setup.py. (It looks like maintainership passed from Michael Foord to Robert Collins over time.)